cli/app/app.go

```diff
@@ -242,6 +228,21 @@ func Down(c *cli.Context) {
 
 }
 
+// Convenience method to return the appropriate Transformer based on
+// what provider we are using.
+func getTransformer(opt kobject.ConvertOptions) transformer.Transformer {
```

Comment:
can be this moved to transformer package?
There is `GetLoader` fce that is inside loader package, it might be better to have it in the same way.